storm-client/src/jvm/org/apache/storm/Config.java

```diff
@@ -859,9 +859,11 @@ public class Config extends HashMap<String, Object> {
     @IsString
     public static final String STORM_DO_AS_USER = "storm.doAsUser";
     /**
-     * The number of machines that should be used by this topology to isolate it from all others. Set storm.scheduler to
-     * org.apache.storm.scheduler.multitenant.MultitenantScheduler
-     */
+     * The number of machines that should be used by this topology to isolate it from all others.
```

Comment:
Unrelated to your change, but hopefully you can address...  I don't understand why specifying a number of machines causes isolation.  Your PR mentions this is a max number of machines.  Can you add/modify the description to clarify this config description?

Reply:
Added more detailed explanation.